Client/Frontend/Onboarding/OnboardingCardViewController.swift

```diff
@@ -193,23 +219,25 @@ class OnboardingCardViewController: UIViewController, Themeable {
     func setupView() {
         view.backgroundColor = .clear
 
-        contentStackView.addArrangedSubview(imageView)
-        contentStackView.addArrangedSubview(titleLabel)
-        contentStackView.addArrangedSubview(descriptionBoldLabel)
-        contentStackView.addArrangedSubview(descriptionLabel)
-        contentContainerView.addSubviews(contentStackView)
+        topStackView.addArrangedSubview(imageView)
+        topStackView.addArrangedSubview(titleLabel)
+        topStackView.addArrangedSubview(descriptionBoldLabel)
+        topStackView.addArrangedSubview(descriptionLabel)
```

Comment:
I see the benefit of using `.build` from [UIViewExtension](https://github.com/mozilla-mobile/firefox-ios/blob/main/BrowserKit/Sources/Common/Extensions/UIViewExtension.swift#L22-L28); however, one downfall is that to save us from writing one line of code (view.translatesAutoresizingMaskIntoConstraints = false), we're forced to add views to a StackView in this way.

I wonder if it would be better to do something like:
```swift
private lazy var topStackView: UIStackView = {
    let stack = UIStackView(withArrangedSubviews: [imageView, titleLabel, descriptionBoldLabel, descriptionLabel]
    stack.translatesAutoresizingMaskIntoConstraints = false
    stack.backgroundColor = .clear
    stack.alignment = .center
    stack.spacing = UX.topStackViewSpacing
    stack.axis = .vertical
    return stack
}()
```

Then, you can remove Lines 222-225.

Reply:
I understand your argument, and it makes sense. However, I think maintaining consistency across the codebase is more important, so that things are predictable and easily consumed at a glance. We know that view setup in the project happens in `setupView()` of any particular ViewController, and if we're trying to debug, it's much faster to go there and look for things, than to start scrolling around looking for different things.

Our team can always revisit this `.build` pattern we're currently using at a later date, and ask whether it's worth pursuing it. But, of course, that has to be balanced against refactoring everything to maintain consistency. Our codebase is slowly being worked on to bring it into shape because for a long time it was the perfect example of spaghetti code, and internal consistency is a key thing to getting things to a solid place. From there, we can look at how we improve things.